### DISCUSSION TITLE
geopandas cython-dask

### DISCUSSION THREAD

**Discussion by mikilterribile** [geopandas/geopandas#3196] [General]
"Hi everyone, while searching the internet to speed up GeoPandas, especially for plotting operations (gdf.plot), I came across the 'GeoPandas Cython' project. However, I'm not clear whether these improvements are already incorporated into the latest version of GeoPandas, or if I need to install GeoPandas Cython or other packages separately. Thanks in advance for your attention. Best regards, Michele Zucchelli."

**Comment by jorisvandenbossche** [comment]
Yes, those improvements have been all integrated already, just not in geopandas itself but first through `pygeos`, and then this was fully integrated into `shapely`. 

Thus, if you have the latest geopandas with latest version of shapely (>=2), then you have all performance improvements enabled. 

(I do have an old blog post about geopandas cython myself, I should probably add a disclaimer to it noting this)

**Comment by mikilterribile** [comment]
Awesome, thanks!

Il ven 23 feb 2024, 11:35 Joris Van den Bossche ***@***.***>
ha scritto:
